features/camel/src/main/feature/feature.xml

```diff
@@ -165,9 +144,11 @@
         <feature version="${cxf-version-range}">cxf-core</feature>
         <feature version="${cxf-version-range}">cxf-jaxrs</feature>
         <feature version="${cxf-version-range}">cxf-jaxws</feature>
+        <!--<feature version="${cxf-version-range}">cxf-http-provider</feature>-->
```

Comment:
Does this need to be commented out? A comment in the file as to why might be good.

Reply:
its what we did before ill add it to the comment block above.